libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -539,6 +552,7 @@ bool AudioIO::StartPortAudioStream(const AudioIOStartStreamOptions &options,
    bool usePlayback = false, useCapture = false;
    PaStreamParameters playbackParameters{};
    PaStreamParameters captureParameters{};
+   PaWasapiStreamInfo wasapiStreamInfo{};
```

Comment:
This type is not known except when compiling for Windows.  So `#ifdef __WXMSW__` and `endif` should surround this line too, and other lines below that do things with this variable.